data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/model/ExportObjectKey.java

```diff
@@ -36,7 +36,9 @@ public static ExportObjectKey fromString(final String objectKeyString) {
         final String prefix = parts[0];
         final String exportTaskId = parts[1];
         final String databaseName = parts[2];
-        final String tableName = parts[3];
+        // fullTableName is in the format of "databaseName.tableName"
+        final String fullTableName = parts[3];
+        final String tableName = fullTableName.split("\\.")[1];
```

Comment:
Should we enforce validation here ? Can there be user error that may lead to NPE ?